File: base/budega/.cache/draft/ts/student.ts
class Adapter {
    // private mercantil: Mercantil = new Mercantil(0);
    init(nCaixas: number) {
        // this.mercantil = new Mercantil(nCaixas);
    }

    arrive(nome: string): void {
        // this.mercantil.chegar(new Pessoa(nome));
    }

    call(index: number): void {
        // this.mercantil.chamar(index);
    }

    finish(index: number): void {
        // this.mercantil.finalizar(index);
    }

    show(): void {
        // console.log(this.mercantil.toString());
    }
}

if (require.main === module) {
    console.log("Rodando o arquivo do estudante");
}

export { Adapter };

File: base/budega/.cache/draft/java/Student.java
class Adapter {
    // private Mercantil mercantil = new Mercantil(0);
    public void init(int qtd_caixas) {
        // this.mercantil = new Mercantil(qtd_caixas);
    }

    public void arrive(String name) {
        // this.mercantil.chegar(new Pessoa(name));
    }
    public void call(int indice) {
        // this.mercantil.chamarNoCaixa(indice);
    }
    public void finish(int indice) {
        // this.mercantil.finalizar(indice);
    }

    public void show() {
        // System.out.println(this.mercantil.toString());
    }
}

public class Student {
    public static void main(String args[]) {
        System.out.println("Rodando o arquivo do estudante");
    }
}